service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/AllocationDecisionController.java

```diff
@@ -59,4 +72,34 @@ public AboutToStartOrSubmitCallbackResponse checkIfAllocationProposalIsMissing(
     private String checkIfAllocationProposalIsPresent(AllocationProposal data) {
         return data != null && StringUtils.isNotEmpty(data.getProposal()) ? "Yes" : "No";
     }
+
+    private String checkIfAllocationDecisionIsPresent(AllocationDecision data) {
+        return data != null && StringUtils.isNotEmpty(data.getProposal()) ? "Yes" : "No";
```

Comment:
Something for the future, but I am happy to accept this and we can refactor in the future - you could write interface which have got getProposal method so you can have only one checkIf method.

Reply:
as discussed with you this will be future refactor :)